NEWS

```diff
@@ -1,3 +1,33 @@
+libmongoc 1.20.0
```

Comment:
The name change of mongo-c-driver to libmongoc comes from this slack conversation: https://mongodb.slack.com/archives/GMY98JS86/p1633468236124400. That change is reflected in the latest version of the release script.